src/test/java/com/crud/tasks/trello/facade/TrelloFacadeTestSuite.java

```diff
@@ -1,65 +1,123 @@
-import static org.mockito.ArgumentMatchers.anyList;
-import static org.mockito.Mockito.when;
+import static org.junit.Assert.assertEquals;
 
 @RunWith(MockitoJUnitRunner.class)
 public class TrelloFacadeTestSuite {
 
     @InjectMocks
-    private TrelloFacade trelloFacade;
-
-    @Mock
     private TrelloMapper trelloMapper;
 
-    @Mock
-    private TrelloValidator trelloValidator;
+    @Test
+    public void testMapToBoard() {
+        //Given
+        TrelloListDto trelloListDto = new TrelloListDto("1", "trello list dto", false);
+        List<TrelloListDto> trelloList = new ArrayList<>();
+        trelloList.add(trelloListDto);
+
+        TrelloBoardDto trelloBoard = new TrelloBoardDto("board_dto", "1", trelloList);
+        List<TrelloBoardDto> boardDtoList = new ArrayList<>();
+        boardDtoList.add(trelloBoard);
 
-    @Mock
-    private TrelloService trelloService;
+        //When
+        List<TrelloBoard> mappedList = trelloMapper.mapToBoard(boardDtoList);
+
+        //Then
+        assertEquals(boardDtoList.get(0).getName(), mappedList.get(0).getName());
+        assertEquals(boardDtoList.get(0).getId(), mappedList.get(0).getId());
+        assertEquals(boardDtoList.get(0).getLists().get(0).isClosed(), mappedList.get(0).getLists().get(0).isClosed());
+    }
 
     @Test
-    public void testFetchEmptyList() {
+    public void testMapToBoardDto() {
         //Given
+        TrelloList trelloList = new TrelloList("1", "trello list dto", true);
+        List<TrelloList> trelloLists = new ArrayList<>();
+        trelloLists.add(trelloList);
+
+        TrelloBoard trelloBoard = new TrelloBoard("2", "trello board", trelloLists);
+        List<TrelloBoard> trelloBoards = new ArrayList<>();
+        trelloBoards.add(trelloBoard);
+
+        //When
+        List<TrelloBoardDto> mappedList = trelloMapper.mapToBoardDto(trelloBoards);
+
+        //Then
+        Assert.assertEquals(trelloBoards.get(0).getId(),mappedList.get(0).getId());
+        Assert.assertEquals(trelloBoards.get(0).getName(),mappedList.get(0).getName());
+        Assert.assertEquals(trelloBoards.get(0).getLists().get(0).isClosed(),mappedList.get(0).getLists().get(0).isClosed());
+    }
+
+    @Test
+    public void testMapToList() {
+        //Given
+        TrelloListDto trelloListDto = new TrelloListDto("1", "trello list dto", false);
         List<TrelloListDto> trelloList = new ArrayList<>();
-        trelloList.add(new TrelloListDto("1", "list1", false));
+        trelloList.add(trelloListDto);
 
-        List<TrelloBoardDto> trelloBoard = new ArrayList<>();
-        trelloBoard.add(new TrelloBoardDto("1", "board_dto", trelloList));
+        //When
+        List<TrelloList> mappedList = trelloMapper.mapToList(trelloList);
 
-        List<TrelloList> mappedTrelloList = new ArrayList<>();
-        mappedTrelloList.add(new TrelloList("1", "list1", false));
+        //Then
+        Assert.assertEquals(trelloList.get(0).getId(), mappedList.get(0).getId());
```

Comment:
czy te asercje na pewno pokrywają oczekiwania co do testu? Brakuje asercji co do rozmiaru listy. moze poszukaj, jak uzyc asercji AssertJ do sprawdzenia, ze dwie listy sa rowne (bo o to tu chodzi)

Reply:
Myślę, że pokrywają. Dodałem nowe asercje ale wydaję mi się że cześć się pokrywa.